.github/workflows/build.yml

```diff
@@ -50,6 +51,22 @@ jobs:
   build_macos:
     name: 'macOS'
     runs-on: macos-latest
+    env:
+      PYTHONOLDPARSER: old
+    steps:
+    - uses: actions/checkout@v1
+    - name: Configure CPython
+      run: ./configure --with-pydebug --with-openssl=/usr/local/opt/openssl --prefix=/opt/python-dev
+    - name: Build CPython
+      run: make -j4
+    - name: Display build info
+      run: make pythoninfo
+    - name: Tests
+      run: make buildbottest TESTOPTS="-j4 -uall,-cpu"
```

Comment:
Is it really worth it to slow down the pre-commit to test the old parser on different platforms? A single pre-commit job would be enough, no? (Pick your favorite platform.) Or maybe only post-commit buildbots would be enough.